hermes-api/src/main/java/pl/allegro/tech/hermes/api/SubscriptionPolicy.java

```diff
@@ -13,44 +13,60 @@
     private static final int DEFAULT_RATE = 400;
     private static final int DEFAULT_MESSAGE_TTL = 3600;
     private static final int DEFAULT_MESSAGE_BACKOFF = 100;
+    private static final int DEFAULT_REQUEST_TIMEOUT = 1000;
+    private static final int DEFAULT_INFLIGHT_SIZE = 100;
```

Comment:
What with the property CONSUMER_INFLIGHT_SIZE, is it still used anywhere? Shall we remove it as well or use the CONSUMER_INFLIGHT_SIZE and the removed CONSUMER_HTTP_CLIENT_REQUEST_TIMEOUT as defaults instead of hardcoding them here?